metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/indexing_commands.py

```diff
@@ -94,24 +96,24 @@ def init_hdfs_dir(self):
         Logger.info('Done creating HDFS indexing directory')
 
     def start_indexing_topology(self):
-        Logger.info("Starting Metron indexing topology: {0}".format(self.__indexing))
+        Logger.info("Starting Metron indexing topology: {0}".format(self.__indexing_topology))
         start_cmd_template = """{0}/bin/start_elasticsearch_topology.sh \
                                     -s {1} \
                                     -z {2}"""
-        Logger.info('Starting ' + self.__indexing)
+        Logger.info('Starting ' + self.__indexing_topology)
         if self.__params.security_enabled:
             metron_security.kinit(self.__params.kinit_path_local,
                                   self.__params.metron_keytab_path,
                                   self.__params.metron_principal_name,
                                   execute_user=self.__params.metron_user)
-        Execute(start_cmd_template.format(self.__params.metron_home, self.__indexing, self.__params.zookeeper_quorum),
+        Execute(start_cmd_template.format(self.__params.metron_home, self.__indexing_topology, self.__params.zookeeper_quorum),
                 user=self.__params.metron_user)
 
         Logger.info('Finished starting indexing topology')
 
     def stop_indexing_topology(self):
-        Logger.info('Stopping ' + self.__indexing)
-        stop_cmd = 'storm kill ' + self.__indexing
+        Logger.info('Stopping ' + self.__indexing_topology)
```

Comment:
I just want to make sure I understand the changes you made here.  Seems like you fixed a bug, but just want to be sure.